manifests/charts/base/templates/services.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  {{- if .Values.pilot.enabled }}
```

Comment:
This also really doesn't belong in `base` - it is always revisioned and it should just go into `istio-discovery`, I think. I'll pick this back up in https://github.com/istio/istio/pull/53016